pkg/broker/broker.go

```diff
@@ -769,8 +770,38 @@ func (a AnsibleBroker) isJobInProgress(instance *apb.ServiceInstance,
 	return len(methodJobs) > 0, token, nil
 }
 
+func (a AnsibleBroker) GetBind(instance apb.ServiceInstance, bindingUUID uuid.UUID) (*BindResponse, error) {
+
+	log.Debug("XXX entered GetBind")
```

Comment:
Is 'XXX' suppose to be substituted for something? Or is this to log that "we're running GetBind"?